styles.css

```diff
@@ -7,12 +7,18 @@
 body {
   margin: 0;
   padding: 0;
+  font-family: 'Source Sans Pro', sans-serif;
 }
 
 header {
     background: black;
 }
 
+h1, h2 {
```

Comment:
Same as above: I'm not sure why we're not getting a merge conflict here, but since you already added this line in another PR you can remove it from this one.

Reply:
Ok I'll make those changes.